src/Entities/Player.cpp

```diff
@@ -209,6 +213,15 @@ void cPlayer::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		}
 	}
 
+	Vector3d CurrPos = GetPosition();
+	std::chrono::steady_clock::duration timeDiff = std::chrono::steady_clock::now() - m_LastTickTime;
```

Comment:
If you're going to calculate the value every tick then the a_Dt argument to this method will be equal to timeDiff, but with millisecond precision.